components/quickbooks/sources/new-or-modified-customer/new-or-modified-customer.js

```diff
@@ -0,0 +1,52 @@
+const quickbooks = require("../../quickbooks.app");
+const common = require("../common");
+
+const sourceEntity = "Customer";
+
+const supportedOperations = common.methods.getSupportedOperations(sourceEntity);
+
+module.exports = {
+  ...common,
+  key: "quickbooks-new-or-modified-customer",
+  name: "New or Modified Customer (Created, Updated, Merged or Deleted)",
```

Comment:
The [current pattern](https://pipedream.com/docs/components/guidelines/#source-name) to identify a real-time event source is to include “(Instant)” in the source name. E.g., “New Search Mention” or “New Submission (Instant)”.

```suggestion
  name: "New or Modified Customer (Created, Updated, Merged or Deleted) (Instant)",
```